redhat-certification/psmdb/images-preparation.bash

```diff
@@ -0,0 +1,59 @@
+repo='percona'
+psmdb-operator="percona-server-mongodb-operator"
+psmdb="percona-server-mongodb"
+pbm="percona-backup-mongodb"
+release='1.16.0'
+pmm_version='2.41.2'
+psmdb7_version="7.0.8-5"
+psmdb6_version="6.0.15-12"
+psmdb5_version="5.0.26-22"
+psmdb_backup_version="2.4.1"
+
+export PSMDB_IMAGES_PROJECT_ID="5e627846b6bf136294e8bb8b"
+export PSMDB_OPERATOR_PROJECT_ID="5e62470102235d3f505f60e3"
+export PSMDB_IMAGES_PROJECT_USER="redhat-isv-containers+5e627846b6bf136294e8bb8b-robot"
+export PSMDB_OPERATOR_PROJECT_USER="redhat-isv-containers+5e62470102235d3f505f60e3-robot"
```

Comment:
Are these private?
Also ,we can simplify to `PSMDB_IMAGES_PROJECT_USER="redhat-isv-containers+$PSMDB_IMAGES_PROJECT_ID-robot"` and the same for the operator